### DISCUSSION TITLE
sb Adguard，拦我页面

### DISCUSSION THREAD

**Discussion by lixiny93** [qier222/YesPlayMusic#2072] [General]
开了adguard就没消停过，部署完打开，卡加载
看控制台是看报错真的迷惑，a b c d e....26字母凑全了，根本看不出哪报错了
还以为是我部署错了，改vercel.json改了又该，依旧没解决
最后看网络发现有资源被拦截了，备注 adguard ，把adguard关了才正常加载
我还以为是firefox的问题，结果是sb广告拦截的问题！

**Comment by Jeamorg** [comment]
既然是自己的网站，广告就不用拦截了吧，而且 YesPlayMusic 也没广告
